libexec/copy-from-target

```diff
@@ -46,13 +46,13 @@ if [ $# = 0 ] ; then
   exit 1
 fi
 
-if [ -n "$USE_DOCKER" ]; then
+if [ "$USE_DOCKER" -eq 1 ]; then
```

Comment:
here and elsewhere, let's use string comparison (i.e. `== "1"`)

Reply:
Fixed, actually script wasn't compiling with it. Didn't see you're using `sh`.